python-lib/gluonts_forecasts/trained_model.py

```diff
@@ -25,7 +28,8 @@ class TrainedModel:
         forecasts_df (DataFrame): Dataframe with the different quantiles forecasts and the training data if include_history is True
     """
 
-    def __init__(self, predictor, gluon_dataset, prediction_length, quantiles, include_history, history_length_limit=None):
+    def __init__(self, model_name, predictor, gluon_dataset, prediction_length, quantiles, include_history, history_length_limit=None):
```

Comment:
Can't the model_name be derived from the `predictor`? Do we need to add it as a mandatory attribute?

Reply:
all the deep learning models predictor are of the same class (gluonts.mx.model.predictor.RepresentableBlockPredictor)
I can make it optional and use the previous behavior (don't remove anything from the list_dataset) when no name is set

Reply:
Okay, let's keep it then, good idea. Closing...

Reply:
done in d42615b